core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -151,6 +168,25 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
     }
   }
 
+  private def startMonitoringNewSubmissions(): Unit = {
+    datasource.inTransaction { dataAccess =>
+      dataAccess.submissionQuery.listAllActiveSubmissions() map { activeSubs =>
+        val monitoredSubmissions = context.children.map(_.path.name).toSet
+        val unmonitoredSubmissions = activeSubs.filterNot(sub => monitoredSubmissions.contains(sub.submission.submissionId))
+
+        unmonitoredSubmissions.map { activeSub =>
+          val wsName = WorkspaceName(activeSub.workspaceNamespace, activeSub.workspaceName)
+          val subId = activeSub.submission.submissionId
+
+          self ! SubmissionStarted(wsName, UUID.fromString(subId))
+        }
+      }
+    } onFailure {
+      case t: Throwable => logger.error("Error starting submission monitor actors for new submissions", t)
+    }
+    self ! SubmissionMonitorPassComplete
```

Comment:
this last line executes outside the inTransaction future, so this is actually a lie. The monitor pass is not complete.

A better pattern is for this function to return `Future[SubmissionMonitorPassComplete]` and then the receive function would be like ```case StartMonitorPass => startMonitoringNewSubmissions() pipe self```